### DISCUSSION TITLE
the default behavior in the bookmark treeview could be customed by user？

### DISCUSSION THREAD

**Discussion by charygao** [sumatrapdfreader/sumatrapdf#4121] [General]
1. when I press ‘→’ button to pagedown the pdf content，the bookmark index for the current page while auto selected and display in the bookmark treeview bottom by auto scrolling。 see the pic:
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/6499936/1be04e8f-070e-42e7-bde8-63448a6bbeef)
2.but I wish the bookmark index for the current page while auto selected and  **auto display in the vertical middle center position**  for the bookmark treeview window。so that I will see the next paragraph title to understand the curent content indeeper。
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/6499936/b5b8e147-f1a3-4e81-b9e2-fe3a1c034405)
3. please add the settings for user: TreeViewAutoScollPosition='top/middle/bottom'  \ TreeViewAutoScollPosOffset='+1/0/-1'
thank you!

**Comment by kjk** [comment]
Please use issues for bug reports and features request.

That being said: I don't think it's possible. We use Windows' TreeView control and https://learn.microsoft.com/en-us/windows/win32/controls/tvm-selectitem message to implement this functionality.

The only options that I see available are to show it at end (what we do) or at the top (with `TVGN_FIRSTVISIBLE` flag).

There's no option for the middle and between top / bottom option, bottom is less bad option.
